### DISCUSSION TITLE
Badge

### DISCUSSION THREAD

**Discussion by mimarz** [digdir/designsystemet#1815] [Components]
Component requested by Dialogporten.

The need is for a component to indicate a number of notifications using numerical values that can be attached or inlined with other elements.

Things to consider:
- Need to review this against usage with `Tag`
- Is this just `Tag` with rounded borders or a separate component? Mixed definitions out here

Examples
- https://atlassian.design/components/badge/examples
- https://orbit.kiwi/components/information/notificationbadge/
- 
<img width="375" alt="Skjermbilde 2024-04-13 kl  10 34 09" src="https://github.com/digdir/designsystemet/assets/33222679/709e0988-75da-4fa1-b82d-c4c59c182f36">

**Comment by mimarz** [comment]
If we get the time we could look at this before v1, or else after.

Should be done in conjunction with #1092

**Comment by Barsnes** [comment]
Tag vs Badge: 
> Badges and tags are visually similar, but tags are interactive and badges are not. 

ref: https://www.delldesignsystem.com/components/badge/

**Comment by mrosvik** [comment]
The component is created and published: https://next.storybook.designsystemet.no/?path=/docs/komponenter-badge--docs
